components/Shared/Header.js

```diff
@@ -158,3 +161,14 @@ const attach = e => {
   document.body.classList.remove('box-collapse-closed')
 }
 
+const changeIcon = () =>{
+  const query = document.getElementById('searchInputHeader');
+  let search_button = document.getElementById('search_button')
+  if(query.value.length <= 1){
```

Comment:
I think this should just be `if(query.value.length < 1)`

Reply:
Yes I used `<= 1` because inside the `attach` function the condition for showing the advance search box is set as `query.value.length>1` So I thought this is a required thing so as to promote user to enter at least two characters.

Reply:
Alright! Sounds fine to me.